code.js

```diff
@@ -0,0 +1,368 @@
+   iconEnter.className = 'fas fa-pen iconEnter';
+   listItemTitle.appendChild(document.createTextNode('To Do List'));
+   listItemTitle.appendChild(iconRefresh);
+   const listItemInput = document.createElement('li');
+   listItemInput.className = 'listItemInput';
+   const listInput = document.createElement('input');
+   listInput.type = 'text';
+   listInput.className = 'listInput';
+   listInput.id = 'listInput';
+   listInput.addEventListener('keyup', addItem, false);
+   listInput.placeholder = 'Add to your list';
+   listItemInput.appendChild(iconEnter);
+   listItemTitle.className = 'listItem listItemTitle';
+   listItemInput.appendChild(listInput);
+   listMain.appendChild(listItemTitle);
+   listMain.appendChild(listItemInput);
+   this.listItems.forEach((item) => {
+     const listItem = document.createElement('li');
+     const checkbox = document.createElement('input');
+     checkbox.type = 'checkbox';
+     checkbox.name = `checkbox-${item.index - 1}`;
+     checkbox.value = 'value';
+     checkbox.id = `checkbox-${item.index - 1}`;
+     checkbox.className = 'checkbox';
+     if (item.completed === true) {
+       const thickSyb = document.createElement('span');
+       thickSyb.className = 'thickSyb';
+       thickSyb.addEventListener('click', unCheckComplete, false);
+       const t = document.createTextNode('✔ ');
+       thickSyb.appendChild(t);
+       listItem.appendChild(thickSyb);
+     } else {
+       checkbox.checked = false;
+       listItem.appendChild(checkbox);
+     }
+     listItem.id = item.index;
+     listItem.draggable = true;
+     listItem.className = 'listItem listItemDrag';
+     listItem.addEventListener('click', activeList, false);
+
+     const textDesc = document.createElement('span');
+     textDesc.className = 'textDesc';
+     const t = document.createTextNode(item.decription);
+     textDesc.appendChild(t);
+     listItem.appendChild(textDesc);
+
+     const listInputDsec = document.createElement('input');
+     listInputDsec.type = 'text';
+     listInputDsec.className = 'listInputDsec hide';
+     listInputDsec.addEventListener('keyup', updateItem, false);
+     listInputDsec.placeholder = 'Add to your list';
+
+     listItem.appendChild(listInputDsec);
+
+     const spanSvg = document.createElement('span');
+     spanSvg.className = 'spanSvg';
+     const icon = document.createElement('svg');
+     icon.id = `icon-${item.index - 1}`;
+     icon.className = 'fas fa-ellipsis-v icon';
+     spanSvg.addEventListener('click', removeItem, false);
+     spanSvg.appendChild(icon);
+     listItem.appendChild(spanSvg);
+     listMain.appendChild(listItem);
+   });
+   if (this.listItems !== []) {
+     const clearLink = document.createElement('li');
+     clearLink.className = 'listItem clearLink';
+     clearLink.appendChild(document.createTextNode('Clear all completed'));
+     clearLink.addEventListener('click', clearComplted, false);
+     listMain.appendChild(clearLink);
```

Comment:
Optional: There are repetitive codes in element creation of elements. It would be great if you could define a function that simplifies the creation of these elements.